pkg/service/presentation/storage/storage.go

```diff
@@ -0,0 +1,39 @@
+package storage
+
+import (
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/tbd54566975/ssi-service/internal/util"
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+)
+
+type StoredPresentation struct {
+	ID                     string                          `json:"id"`
+	PresentationDefinition exchange.PresentationDefinition `json:"presentation_definition"`
```

Comment:
camelCase

Reply:
agree here, we only need to use snake_case as it is required by the specs in our API boundaries. internal to the system we should use camel. similar to - https://github.com/TBD54566975/ssi-service/blob/main/pkg/service/manifest/storage/storage.go#L12